userspace/libsinsp/container.cpp

```diff
@@ -232,18 +232,11 @@ string sinsp_container_manager::container_to_json(const sinsp_container_info& co
 
 bool sinsp_container_manager::container_to_sinsp_event(const string& json, sinsp_evt* evt, shared_ptr<sinsp_threadinfo> tinfo)
 {
-	// TODO: variable event length
-	size_t evt_len = SP_EVT_BUF_SIZE;
 	size_t totlen = sizeof(scap_evt) +  sizeof(uint16_t) + json.length() + 1;
 
-	if(totlen > evt_len)
-	{
-		g_logger.format(sinsp_logger::SEV_ERROR,
-				"container_to_sinsp_event: event len %d > max len %d w/ json \"%s\", returning false",
-				totlen, evt_len, json.c_str());
-		ASSERT(false);
-		return false;
-	}
+	ASSERT(evt->m_pevt_storage == nullptr);
+	evt->m_pevt_storage = new char[totlen];
```

Comment:
minor nit: since totlen can now be arbitrarily large, would it make sense to check for null return from new and bail instead of crashing later when m_pevt_storage is accessed?